k8s/safe/kustomizations/local/photon/Deployment.yaml

```diff
@@ -0,0 +1,120 @@
+#
+# .-'_.---._'-.
+# ||####|(__)||   Protect your secrets, protect your business.
+#   \\()|##//       Secure your sensitive data with Aegis.
+#    \\ |#//                    <aegis.ist>
+#     .\_/.
+#
+
+apiVersion: apps/v1
```

Comment:
Since the `-k` flag of `kubectl` expects a directory, and since Kustomize wants all the resources to be in child folders (and not in parent folders for security reasons), the easiest way to split different customizations was to copy the same `Deployment.yaml` to each folder separately.

It’s a bit repetitive, but results in a cleaner folder structure, and also gives the flexibility to customize deployments further when/if there is a need.

For example, we can add a label for each deployment, etc.